cloudfront/iiif.wellcomecollection.org/terraform/cloudfront_distro/main.tf

```diff
@@ -122,244 +53,37 @@ resource "aws_cloudfront_distribution" "iiif" {
-
-    forwarded_values {
-      query_string = true
-      headers      = []
-
-      cookies {
-        forward = "none"
-      }
-    }
-
-    dynamic "lambda_function_association" {
-      for_each = var.dlcs_lambda_associations
-      content {
-        event_type = lambda_function_association.value["event_type"]
-        lambda_arn = lambda_function_association.value["lambda_arn"]
-      }
-    }
-
-    min_ttl     = 604800
-    default_ttl = 86400
-    max_ttl     = 31536000
-
-    viewer_protocol_policy = "redirect-to-https"
-  }
-
-  ordered_cache_behavior {
-    path_pattern     = "image/N00*"
-    allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
-    cached_methods   = ["GET", "HEAD", "OPTIONS"]
-    target_origin_id = var.miro_sourced_images_target
-
-    forwarded_values {
-      query_string = true
-      headers      = []
-
-      cookies {
-        forward = "none"
-      }
-    }
-
-    dynamic "lambda_function_association" {
-      for_each = var.dlcs_lambda_associations
-      content {
-        event_type = lambda_function_association.value["event_type"]
-        lambda_arn = lambda_function_association.value["lambda_arn"]
-      }
-    }
-
-    min_ttl     = 604800
-    default_ttl = 86400
-    max_ttl     = 31536000
-
-    viewer_protocol_policy = "redirect-to-https"
-  }
-
-  ordered_cache_behavior {
-    path_pattern     = "image/A00*"
-    allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
-    cached_methods   = ["GET", "HEAD", "OPTIONS"]
-    target_origin_id = var.miro_sourced_images_target
-
-    forwarded_values {
-      query_string = true
-      headers      = []
-
-      cookies {
-        forward = "none"
-      }
-    }
-
-    dynamic "lambda_function_association" {
-      for_each = var.dlcs_lambda_associations
-      content {
-        event_type = lambda_function_association.value["event_type"]
-        lambda_arn = lambda_function_association.value["lambda_arn"]
-      }
-    }
-
-    min_ttl     = 604800
-    default_ttl = 86400
-    max_ttl     = 31536000
-
-    viewer_protocol_policy = "redirect-to-https"
-  }
-
-  ordered_cache_behavior {
-    path_pattern     = "image/W00*"
-    allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
-    cached_methods   = ["GET", "HEAD", "OPTIONS"]
-    target_origin_id = var.miro_sourced_images_target
-
-    forwarded_values {
-      query_string = true
-      headers      = []
-
-      cookies {
-        forward = "none"
+  dynamic "ordered_cache_behavior" {
+    for_each = var.behaviours
+    content {
+      path_pattern     = ordered_cache_behavior.value["path_pattern"]
+      allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+      cached_methods   = ["GET", "HEAD", "OPTIONS"]
+      target_origin_id = ordered_cache_behavior.value["target_origin_id"]
+
+      forwarded_values {
+        query_string = true
```

Comment:
Do we use the query string?

Reply:
I don't actually think we do, but this is the same as currently configured, so i'm reluctant to fix it in this PR.